scripts/go.mod

```diff
@@ -14,3 +14,5 @@ require (
 	gopkg.in/check.v1 v1.0.0-20180628173108-788fd7840127 // indirect
 	gopkg.in/yaml.v2 v2.2.2 // indirect
 )
+
+go 1.13
```

Comment:
Is this true of Beats as well? Not sure if this will cause issues

Reply:
The `make` process did this, but I can revert that particular file if will cause issues.